api/test/wfl/integration/modules/covid_test.clj

```diff
@@ -0,0 +1,67 @@
+(ns wfl.integration.modules.covid-test
+  (:require [clojure.test :refer :all]
+            [clojure.test :as clj-test]
+            [wfl.tools.fixtures :as fixtures]
+            [wfl.jdbc :as jdbc]
+            [wfl.module.covid :as covid]
+            [wfl.service.rawls :as rawls])
+  (:import [clojure.lang ExceptionInfo]))
+
+(clj-test/use-fixtures :once fixtures/temporary-postgresql-database)
+
+(def workload {:id 1})
```

Comment:
All of these indiv. `def`s have me thinking that there must be a better way to set up values for testing.  Maybe an `.edn` file?

I'm inclined to wait for the data model to stabilize and see how others use testing data.

Reply:
I wouldn't worry too much about it for now. This is easy enough to grep though and change as needed.
I generally move constants shared between tests into the test tools but we can refactor this later.

Reply:
I wouldn't worry too much about it for now. This is easy enough to grep though and change as needed.
I generally move constants shared between tests into the test tools but we can refactor this later.